examples/controls/src/app.js

```diff
@@ -42,17 +49,17 @@ class Root extends Component {
 
   _updateViewport = (viewport) => {
     this.setState({viewport});
-  };
+  }
 
   _resize = () => {
     this.setState({
       viewport: {
         ...this.state.viewport,
         width: window.innerWidth,
-        height: window.innerHeight
+        height: window.innerHeight,
```

Comment:
Nit: How did trailing commas make it into our `.eslintrc`? It's a pretty big break from our coding standard, is it important enough?